index.html

```diff
@@ -2,11 +2,14 @@
 <html lang="en">
 <head>
   <meta charset="UTF-8">
-  <title>Document</title>
+  <title>Exercism v3</title>
   <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
   <meta name="description" content="Description">
   <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
   <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/docsify-themeable@0/dist/css/theme-simple.css">
+
+  <!-- Render triple backtick mermaid blocks as a mermaid graph -->
+  <script src="//unpkg.com/mermaid@8.4.6/dist/mermaid.min.js"></script>

```

Comment:
Can this move down to below `<div id="app"></div>"` pls.

Reply:
Do JS libraries not belong in the `<head>` typically?

Reply:
@SaschaMann It depends. The thing is that if it is in the `<head>`, it will potentially block loading the rest of the page.

Reply:
> Do JS libraries not belong in the <head> typically?

As Erik says it depends, but my strong opinion is that basically they never should. So on Exercism, all JS loads after the rest of the page - that means that the site can be rendering text, images, stylesheets, fonts, etc before the JavaScript starts downloading and kicking in. There are special directives you can use on the <script> tag to make JavaScript defer itself until that happens, but they're browser-dependent.

If you're interested in more, I think this article is one of the best out there for the current situation: https://flaviocopes.com/javascript-async-defer

Reply:
Thanks for explaining. I moved it.